### ISSUE TITLE
New Command: Toggle Editor Commenting not showing.

### ISSUE THREAD

**Issue by shyney7** [microsoft/vscode#154768]
Issue Type: <b>Bug</b>

The new command to toggle Editor Commenting cant be found by VSCode.

VS Code version: Code 1.69.0 (92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9, 2022-07-07T05:28:36.503Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No
Remote OS version: Linux x64 4.19.135-cip31-rt13+ind1

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.78GB (8.86GB free)|
|Process Argv|--crash-reporter-id 3f370137-08f1-41c7-b39e-3e75c8b12fc4|
|Screen Reader|no|
|VM|44%|

|Item|Value|
|---|---|
|Remote|SSH: LinuxRemote|
|OS|Linux x64 4.19.135-cip31-rt13+ind1|
|CPUs|Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (12 x 1100)|
|Memory (System)|31.27GB (22.77GB free)|
|VM|0%|
</details><details><summary>Extensions (24)</summary>

Extension|Author (truncated)|Version
---|---|---
QML|bbe|1.0.0
qml|cut|1.3.3
remotehub|Git|0.36.0
better-cpp-syntax|jef|1.15.19
jupyter-keymap|ms-|1.0.0
remote-containers|ms-|0.241.3
remote-ssh|ms-|0.82.1
remote-ssh-edit|ms-|0.80.0
remote-wsl|ms-|0.66.3
remote-repositories|ms-|0.14.0
material-icon-theme|PKi|4.19.0
vscode-icons|vsc|11.13.0
vsconan|afr|0.3.0
doxdocgen|csc|1.4.0
better-cpp-syntax|jef|1.15.19
python|ms-|2022.10.0
vscode-pylance|ms-|2022.7.20
jupyter|ms-|2022.6.1001902341
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.8
cmake-tools|ms-|1.11.26
cpptools|ms-|1.10.8
cpptools-extension-pack|ms-|1.2.0
cmake|twx|0.0.17

(5 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonvspyt551cf:30345471
pythonptprofiler:30281270
vsdfh931cf:30280410
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
vscscmwlcmt:30465135
cppdebug:30492333
vsclangdc:30486549

```

</details>

**Comment by ArturoDent** [comment]
E.g., see https://stackoverflow.com/questions/72356622/multiline-comments-in-vscode-v1-67-resolved-pull-request-comments/72356811#72356811

**Comment by alexr00** [comment]
Thanks @gjsjohnmurray and @ArturoDent, you are both correct. You must first have a workspace with comments + an extension installed that provides comments/commenting ranges. Once you have that, the command will show.
